README.rst

```diff
@@ -109,6 +109,12 @@ Optional arguments:
   be raised if the file can't be read.
   Set to ``False`` to disable this behaviour.
 
+* ``sock`` -- an open socket or socket-like object (such as a `paramiko.Channel`) to use
+  for communication to the target host.  One could setup a proxy_command using this :
```

Comment:
Could you remove the space before the colon? (Also no need for double-spaces after full stops, but it doesn't make any difference to the output.)

Also, is `proxy_command` referring to something specific? It looks like a variable, but isn't referenced anywhere else.

It might also be worth mentioning `ProxyCommand` along with `Channel`, with appropriate links to the Paramiko docs.